package.json

```diff
@@ -49,7 +49,7 @@
     "sass": "^1.26.10",
     "sass-loader": "^9.0.3",
     "style-loader": "^1.2.1",
-    "throttle-debounce": "1.0.1",
+    "throttle-debounce": "^2.2.1",
```

Comment:
Since we are already using lodash, is there any reason to import this package?

Reply:
no,  this is just the previous website's dependency, we will remove it later.

Reply:
OK，I suggest that we can make a refactor todo list.